### PR TITLE
Nazar/teature/settings view v2 change password

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: frontend/src/components/CustomBtn/CustomBtn.module.css [hunk 1]
```diff
@@ -14,8 +14,4 @@
     font-weight: 400;
     font-size: 1.1rem;
     line-height: 1.5rem;
-}
-
-.btnStyle:hover {
-    background-color: #C1E1FF;
 }
```
Comment (Taraslysak) [inline frontend/src/components/CustomBtn/CustomBtn.module.css:17]: EOF line

File: frontend/src/components/CustomBtn/CustomBtn.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import style from "./CustomBtn.module.css";
 type CustomBtnProps = {
   btnName: string;
   additionalStyles?: string;
-  handlerOnClick: () => void;
+  handlerOnClick?: () => void;
```
Comment (Taraslysak) [inline frontend/src/components/CustomBtn/CustomBtn.tsx:7]: button with no onClick prop?

File: frontend/src/pages/ChangePassword/ChangePassword.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        message: "Please provide a valid password.",
+      });
+      console.log(err);
+
+      mutationChangePassword.reset();
+    },
+  });
+
+  const handlerSubmitBtn: SubmitHandler<Inputs> = (data) => {
+    console.log("here1");
```
Comment (Taraslysak) [inline frontend/src/pages/ChangePassword/ChangePassword.tsx:64]: redundant log

**Review by Taraslysak — APPROVED** [review]
Good job!

File: frontend/src/components/CustomBtn/CustomBtn.tsx (hunk 2 above)
Comment (Nazar-kob) [reply frontend/src/components/CustomBtn/CustomBtn.tsx:7]: Yes

File: frontend/src/pages/ChangePassword/ChangePassword.tsx (hunk 3 above)
Comment (Nazar-kob) [reply frontend/src/pages/ChangePassword/ChangePassword.tsx:64]: done

**Review by kipod — APPROVED** [review]
Good

**Pull request merged**